src/utils/pTagRemoval.test.ts

```diff
@@ -0,0 +1,7 @@
+import { pTagRemoval } from "./pTagRemoval";
+
+test("retruns a string without pTags", () => {
```

Comment:
Some of the data has tags which aren't pTags so the tests should cover that, i.e. try an example with anchor tags as well